error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -805,6 +809,20 @@ Flux<T> after(Flux<? extends Iterable<T>> flux, int prefetch) {
     }
   }
 
+  /** Prefer {@link Flux#count()} over collecting into a list and counting its elements. */
+  static final class FluxCount<T> {
```

Comment:
While it's an unwieldy name, likely we'll eventually name this rule `FluxCountMapMathToIntExact` (based on the `@AfterTemplate` code, so let's go for that.